i3bar/src/xcb.c

```diff
@@ -233,10 +243,44 @@ static uint32_t predict_statusline_length(bool use_short_text) {
     return width;
 }
 
+static uint32_t adjust_statusline_length(uint32_t max_length) {
+    uint32_t width = predict_statusline_length();
+
+    /* Progressively switch the blocks to short mode */
+    struct status_block *block;
+    TAILQ_FOREACH (block, &statusline_head, blocks) {
+        if (block->short_text == NULL) {
+            continue;
+        }
+        if (width < max_length) {
+            break;
+        } else if (!block->use_short) {
```

Comment:
```suggestion
        if (width < max_length) {
            break;
        }
        if (block->short_text == NULL || !block->use_short) {
            continue;
        }
        // rest of code without curly brackets
```

One less indentation level.

Reply:
Good call; addressed in 3a4348404d193adbe3d6e08c23a02fc9cbb16112